### ISSUE TITLE
aop-bundle update to 1.2.0 breaks symfony cache:clear

### ISSUE THREAD

**Issue by GWillmann** [schmittjoh/JMSAopBundle#35]
Hi there, 

I don't know what it is related to exactly but updating aop-bundle through *composer update* 
broke our application so we had to force the bundle to stay at version 1.1.0.

Below the error we got : 

<img width="254" alt="screen shot 2015-12-10 at 15 50 15" src="https://cloud.githubusercontent.com/assets/5655900/11718424/26af8d4c-9f56-11e5-88d8-6eaa6bb7e0ef.png">

<img width="848" alt="screen shot 2015-12-10 at 15 50 06" src="https://cloud.githubusercontent.com/assets/5655900/11718433/2eff6da0-9f56-11e5-88c7-b49783734c50.png">

**Comment by perk11** [comment]
Having same issue after updating from 1.1.0 to 1.2.0, everything breaks with this error:
![](http://i.perk11.info/ZSVIGg.png)

**Comment by Ener-Getick** [comment]
It doesn't seem to be related to ``JMSAopBundle``... You can see the changes done in 1.2.0 [here](https://github.com/schmittjoh/JMSAopBundle/compare/1.1.0...master).

**Comment by perk11** [comment]
No, I reverted just AOP and this solved the issue, updated again, and it happens again
